src/networking.c

```diff
@@ -324,7 +326,9 @@ int prepareClientToWrite(client *c) {
 
     /* Schedule the client to write the output buffers to the socket, unless
      * it should already be setup to do so (it has already pending data). */
-    if (!clientHasPendingReplies(c)) putClientInPendingWriteQueue(c);
+    if (!clientHasPendingReplies(c)) {
+        putClientInPendingWriteQueue(c);
+    }
```

Comment:
```suggestion
    if (!clientHasPendingReplies(c)) putClientInPendingWriteQueue(c);
```
Just to keep the code the same.